docs/guides/resources/user-account-resources.primary.in.rst

```diff
@@ -0,0 +1,168 @@
+- Medium (2.0 CPU, 8 GB RAM)
+- Large (4.0 CPU, 16 GB RAM)
+
+Only CPUs (central processing units) are available.
+No GPU (graphical processing units) are available and there is no plan to add them.
+
+- Cores per science user: 0.1 [#f1]_
+
+
+Batch processing
+----------------
+
+This refers to parallel processing (asynchronous) job submission.
+The user batch facility is focused on supporting a large variety of smaller needs for the broader community (`DMTN-202 <https://dmtn-202.lsst.io/>`_).
+User batch processing will be available by Data Release 1 (DR1).
+Access to user batch processing will be allocated by the Resource Allocation Committee.
+
+- Number of core hours total per year: 4.53E+06 [#f2]_
+- Number of core hours per user available via RAC: TBD; to be set by DR1
+
+
+Storage
+=======
+
+During the Early Science era (the Data Previews and Data Release 1), the total amount of shared user space across all user directories (home, project, and scratch) is 24 TB.
+
+During the Operations era, the anticipated amount of individual home directory disk space (in the cloud) is 0.4 TB [#f3]_.
+
+The total shared disk space for batch users approved by the RAC remains TBD.
+
+Backups
+-------
+
+Users are encouraged to use services such as GitHub for software version control and to take care not to accidentally delete files from their home directory.
+
+While there is no guarantee that accidentally deleted data can be recovered, users are encouraged to use the resources for :doc:`/support/index` immediately if mistakes do happen.
+
+Query and memory limits
+=======================
+
+All queries are executed with shared resources.
+The length of time to query completion depends firstly on query design (number of shards accessed), and secondly on number of queries across all users.
+There is no limit on the number of queries a user can do in total (or on daily or yearly timescales), but there are query rate limits.
+
+The size of a dataset retrieved by a query and held in memory depends on the server size which, for the Notebook Aspect, is selected by the user.
+
+- Number of TAP queries per user per 15 minutes: 500 [#f4]_
+- Reset interval after user excession: 15 minutes [#f5]_
+
+
+Portal Aspect TAP (Table Access Protocol) service:
+
+- Maximum rows returned: 5,000,000
+- Maximum table size returned: TBD
+
+Portal Aspect ObsTAP (TAP access to images):
+
+- Maximum rows of image metadata: 5,000,000
+
+Notebook Aspect TAP (Table Access Protocol) service:
+
+- Maximum rows returned: 5,000,000
+- Maximum table size returned: RAM limit of the user-selected server size
+
+Notebook Aspect Butler service:
+
+- Maximum number of simultaneous butler queries per user: TBD (2-5)
+- Maximum number of references returned: no limit
+- Maximum data volume returned: RAM limit of the user-selected server size
+
+API Aspect TAP (Table Access Protocol) service:
+
+- Maximum rows returned: 5,000,000
+- Maximum table size returned: TBD
+
+Download and upload limits
```

Comment:
This is another we-don't-have-a-great-story here, since the RSP does not offer bulk download, and there are different answers for Internet 2 and "public" Internet sites, and also we don't have a recommended mechanism, nor have we estimated throttle values for eg image APIs. If you want this document to go out sooner rather than later you might want to omit this section for now.

Reply:
Left the section header and explanatory text, deleted all the "TBD" rows.